pkg/grpc/api.go

```diff
@@ -327,6 +329,111 @@ func (a *api) DeleteState(ctx context.Context, in *dapr_pb.DeleteStateEnvelope)
 	return &empty.Empty{}, nil
 }
 
+func (a *api) PerformTransaction(ctx context.Context, in *dapr_pb.MultiStateEnvelope) (*empty.Empty, error) {
```

Comment:
@amanbha I would think that a transactional set would just be an attribute in Options and not a new API.

Reply:
@artursouza It would be a new api as its different type of operations on different keys.